api/src/main/java/marquez/db/RunDao.java

```diff
@@ -193,7 +193,7 @@
           + "RETURNING *")
   ExtendedRunRow upsert(
       UUID runUuid,
-      UUID externalId,
+      String externalId,
```

Comment:
Good catch. An `externalId` isn't required to be a `UUID`, but rather a `UUID` is generated for the corresponding `externalId`